packages/dbml-core/__tests__/parser/parser.spec.js

```diff
@@ -10,19 +11,28 @@ describe('@dbml/core', () => {
       const fileExtension = getFileExtension(format);
       const input = require(`./${testDir}/input/${fileName}.in.${fileExtension}`);
       const output = require(`./${testDir}/output/${fileName}.out.json`);
-      const jsonSchema = Parser[parseFuncName](input, format);
+      try {
+        const jsonSchema = Parser[parseFuncName](input);
+        isEqualExcludeTokenEmpty(jsonSchema, output);
+      } catch (error) {
+        if (error instanceof CompilerError) isEqualExcludeTokenEmpty(error.diags, output);
+        else throw error;
+      }
+
 
       /* eslint-enable */
     };
 
     /* eslint-disable */
     test.each(scanTestNames(__dirname, 'dbml-parse/input'))('dbml-parse/%s', (name) => {
-      runTest(name, 'dbml-parse', 'dbml', 'parseDBMLToJSON');
+      runTest(name, 'dbml-parse', 'dbml', 'parseDBMLToJSONv2');
     });
-    
+
+    /*
     test.each(scanTestNames(__dirname, 'mysql-parse/input'))('mysql-parse/%s', (name) => {
       runTest(name, 'mysql-parse', 'mysql', 'parseMySQLToJSONv2');
     });
+    */
```

Comment:
Why are you comment this out?

Reply:
I have updated MySQL test cases and uncommented this block of code nha a